Makefile

```diff
@@ -53,6 +53,8 @@ export USE_CLUSTER_STORAGE ?= true
 export OPERATORS_IN_PRODUCT_NAMESPACE ?= false # e2e tests and createInstallationCR() need to be updated when default is changed
 export DELOREAN_PULL_SECRET_NAME ?= integreatly-delorean-pull-secret
 export ALERTING_EMAIL_ADDRESS = noreply-test@rhmi-redhat.com
```

Comment:
Hey @jjaferson, I might be missing something but should it not be `?=` - currently when specifing the email addresses my envvars are being overwritten by the values in makefile.